test/test_jit.py

```diff
@@ -8558,7 +8558,7 @@ def forward(self, x):
                 return self.ssm.bar() + x
 
         orig = TraceMe()
-        traced = torch.jit.trace(orig, (torch.rand(4, 3, dtype=torch.float),))
+        traced = torch.jit.trace(orig, (torch.rand(4, 3),))
```

Comment:
yeah, this seems like an improvement -- previously we had to mess with the dtype to get this to work right.

Reply:
Yes, but this appears to be restricted to tracing